nlp/README.md

```diff
@@ -0,0 +1,22 @@
+## DACCS - NLP docker image
```

Comment:
I see this README gives instructions to use the Docker image in interactive mode with bash, but I thought this image was used only on the PAVICS platform, using the JupyterHub/JupyterLab setup. 

Is it really in the nlp's team desire to use this image in this way too and is it really something we want to document?
@matprov 

Also, if the new code for nl2query should be usable with JupyterLab via a notebook, it should be tested, and I am wondering if the nl2query folder should be made available to the user (via the tutorial-notebooks folder)?
I guess it could work even if not accessible by the user, if it just a python module that is already installed, but it depends if the nlp team want to have reading access directly on JupyterLab or not.

Reply:
@ChaamC It's not required to document these steps as the actual use case would be through JupyterLab, you're right.
However I find that the simple steps that @ahandan documented are useful in case someone wants to locally test dependency usage locally, before actually building and pushing the image on the docker repo.

Reply:
I agree, this is useful for local tests, so we can keep it then. :)

Still seeing 2 points here :
1. A check on JupyterLab would still be relevant. I would suggest, at first, to maybe just run a local test, by starting the image as you do in the readme, and starting an instance of jupyterlab there, checking if you can use the new dependencies in a notebook. I guess if it works we could then eventually check on staging if nothing breaks (not sure if we want to get into testing using dev setup with OpenStack here?)

2. We should define if the code should be made available to the user on JupyterLab or not, maybe to be defined with nlp team or David B.

Reply:
@ChaamC 
1. Absolutely, the test on an actual instance is a must have, in order to make sure everything is good.
2. What do you mean by "define if the code should be made available to the user on JupyterLab or not"? Since it's packaged inside the image as a dependency, then yes it will be accessible to the user

Reply:
2. I meant if we wanted to include the new files containing code to the JupyterLab environment, in the same way we included some example notebooks files to the tutorial-notebooks folder. But, I think we can leave it as it is, no need to include those files. After all they don't need to look at them like the example notebooks, just to be able to import and use the code. They can refer to the code by going on the repo here if needed.